packages/analytics-js-plugins/src/iubendaConsentManager/constants.ts

```diff
@@ -1,3 +1,3 @@
 const IUBENDA_CONSENT_MANAGER_PLUGIN = 'IubendaConsentManagerPlugin';
-
-export { IUBENDA_CONSENT_MANAGER_PLUGIN };
+const IUBENDA_CONSENT_EXAMPLE_COOKIE_NAME = '_iub_cs-252372';
```

Comment:
Is this the actual cookie name? It is named as "example".

Reply:
yes, the cookie name follows`_iub_cs-${number}`

Reply:
Ok. Why is the number hardcoded in the name?

Reply:
I use this for mocking the cookie in the tests. I updated the util functions tho to get the cookie according to cookie pattern